vault.go

```diff
@@ -481,6 +481,39 @@ func newVaultClient(opts *config) (*api.Client, error) {
 	// step: set the token for the client
 	client.SetToken(token)
 
+	if opts.vaultRenewToken {
+		tokeninfo, err := client.Auth().Token().LookupSelf()
```

Comment:
out of interest is this dependent on a vault version?

Reply:
I don't think so - lookup self has always been around afaik, it just wasn't exposed the same way in the client wrapper. 

https://www.vaultproject.io/api/auth/token/index.html#lookup-a-token-self-